ziti/cmd/edge/quickstart.go

```diff
@@ -65,7 +65,7 @@ func NewQuickStartCmd(out io.Writer, errOut io.Writer, context context.Context)
 	cmd := &cobra.Command{
 		Use:   "quickstart",
 		Short: "runs a Controller and Router in quickstart mode",
-		Long: `runs a Controller and Router in quickstart mode. By default, this will create a totally ephemeral network, only valid while running.`,
+		Long:  "runs a Controller and Router in quickstart mode with a temporary Ziti home directory; suitable for testing and development",
```

Comment:
is the changelog updated for this already? if not, probably worthwhile to update the changelog too